doc/classifying-signatures.md

```diff
@@ -81,12 +81,24 @@ differences between the `sourmash lca` subcommands and the basic
 output structured taxonomic information, and these are what you should look
 to if you are interested in doing classification.
 
+The command `lca gather` applies the `gather` algorithm to search an
+LCA database; it reports taxonomy.
+
 It's important to note that taxonomy based on k-mers is very, very
 specific and if you get a match, it's pretty reliable. On the
 converse, however, k-mer identification is very brittle with respect
 to evolutionary divergence, so if you don't get a match it may only mean
 that the particular species isn't known.
 
+## Abundance weighting
+
+If you compute your input signatures with `--track-abundance`, both
+`sourmash gather` and `sourmash lca gather` will use that information
+to calculate an abundance-weighted result.  Briefly, this will weight
```

Comment:
I think "median kmer multiplicity" should be a column in the output, so users can use that as an estimation of abundance of each genome in metaG. This is more of the feature request. It is all good as basic tutorial here.